docs/dev/reference/changelog.md

```diff
@@ -1230,6 +1230,10 @@ Find this indicator of the time to complete one request/response cycle on your m
 
 The Python SDK now supports Python 3.8, in addition to 3.9 and 3.10.
 
+{{% hiddencontent %}}
+We now support Python 3.11 and later as well.
+{{% /hiddencontent %}}
+
```

Comment:
I would assume this is true
```suggestion
The Python SDK now supports Python 3.8 and newer.
```

Reply:
Yup, this is true. Technically it's 3.8.1. Although 3.8 did reach end of life last year, and we will likely be removing support for it sometime this year.